src/java/org/apache/cassandra/config/Config.java

```diff
@@ -802,6 +802,7 @@ public static void setClientMode(boolean clientMode)
     public volatile DataStorageSpec collection_size_fail_threshold = DISABLED_SIZE_GUARDRAIL;
     public volatile int items_per_collection_warn_threshold = DISABLED_GUARDRAIL;
     public volatile int items_per_collection_fail_threshold = DISABLED_GUARDRAIL;
+    public volatile boolean allow_filtering_enabled = false;
```

Comment:
make this `true`, it should be allowed by default, else its a regression (also yaml says its true)